src/main/java/codechicken/nei/ItemPanel.java

```diff
@@ -105,169 +103,89 @@ public String getRenderLabel() {
                 return ">";
             }
         };
-    }
 
-    public int getX(GuiContainer gui) {
```

Comment:
another public removal

Reply:
Several more below this that I'm not commenting on, but we can't remove public methods that have been exposed.

Reply:
You can check the history though, if they're something I added, please make note of that and it's probably ok to remove as long as you've booted it in the full GTNH pack, because I doubt anyone else would be relying on them.

Reply:
Yes, this method add you.